http/api_handler.go

```diff
@@ -74,6 +74,7 @@ func NewAPIHandler(b *APIBackend) *APIHandler {
 	h := &APIHandler{}
 	b.BucketService = authorizer.NewBucketService(b.BucketService)
 	b.OrganizationService = authorizer.NewOrgService(b.OrganizationService)
+	b.TelegrafService = authorizer.NewTelegrafConfigStore(b.TelegrafService, b.UserResourceMappingService)
```

Comment:
I'd recommend against wrapping it at the top level like this. I did this originally and it caused a bunch of unintended problems. I'd only wrap the specific handle that needs it like

https://github.com/influxdata/influxdb/blob/f889675ef7934df81bd0743c102f680ddd720789/http/api_handler.go#L80

See https://github.com/influxdata/influxdb/pull/11176 for more details